flask_cors/__init__.py

```diff
@@ -229,6 +234,7 @@ def init_app(self, app, **kwargs):
         options.update(_get_app_kwarg_dict(app))
         options.update(self._options)
         options.update(kwargs)
+        logger.debug('setting CORS for app {}'.format(app.name))

```

Comment:
I propose moving this logging statement below the call to _parse_resources, and changing the text slightly:

```Python
logger.debug("Configuring CORS for '%s' with resources: %s", app.name, resources) 
```